openslides/core/apps.py

```diff
@@ -116,6 +116,8 @@ def get_angular_constants(self):
 
         constants: Dict[str, Any] = {}
 
+        raise NotImplementedError()
```

Comment:
This seams wrong

Reply:
ping @FinnStutzenstein

Reply:
> I added an Error, becuase the issue with the users did not come up every time. Please have a look, remove the error, rebase and merge.
So just remove. This was to test this pr.

Reply:
Since I don't have the branch locally anymore, can you do this quickly and alter your commits on this PR?

Reply:
I can do it but not quickly since I'm on my mobile here ;)